msnoise/preprocessing.py

```diff
@@ -15,10 +15,45 @@
 
 from .api import *
 
+# Function returns a pz dict for the total response from the inventory response object
+def get_pazdictfrominventory(inv):
+    polezerostage=inv.get_paz()
+    totalsensitivity=inv.instrument_sensitivity
+    pzdict={}
+    pzdict['poles']=polezerostage.poles
+    pzdict['zeros']=polezerostage.zeros
+    pzdict['gain']=polezerostage.normalization_factor
+    pzdict['sensitivity']=totalsensitivity.value
+    return pzdict
 
 def preprocess(db, stations, comps, goal_day, params):
     datafiles = {}
     output = Stream()
+
```

Comment:
This change will work:
* if you have N inventory files
* if you have 1 and only 1 dataless file

but will not work with N dataless files in the response folder.